### DISCUSSION TITLE
How can I have a URL as plain text?

### DISCUSSION THREAD

**Discussion by anatolykopyl** [facebook/docusaurus#9984] [Q&A] [answered]
Sometimes a URL is not supposed to be clickable. Is there a way to have it be as plain text instead of an anchor?

[Example codesandbox](https://codesandbox.io/p/devbox/modest-jepsen-333zk9?file=%2Fdocs%2Fintro.md%3A15%2C20&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clu8epe4z00083d5wu450k6ng%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clu8epe4z00033d5wt1fryn3r%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clu8epe4z00053d5wpqy2p2r2%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clu8epe4z00073d5w02l4yilb%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clu8epe4z00033d5wt1fryn3r%2522%253A%257B%2522id%2522%253A%2522clu8epe4z00033d5wt1fryn3r%2522%252C%2522activeTabId%2522%253A%2522clu8ernvs004m3d5us3uqa30s%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252F.codesandbox%252Ftasks.json%2522%252C%2522id%2522%253A%2522clu8ernvs004m3d5us3uqa30s%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522clu8epe4z00073d5w02l4yilb%2522%253A%257B%2522id%2522%253A%2522clu8epe4z00073d5w02l4yilb%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clu8epe4z00063d5w2jbl10dj%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522path%2522%253A%2522%252F%2522%257D%252C%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522port%2522%253A3001%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522clu8eug1d00oy3d5ucgmnd7g7%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clu8eug1d00oy3d5ucgmnd7g7%2522%257D%252C%2522clu8epe4z00053d5wpqy2p2r2%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clu8epe4z00043d5wobnrq3u4%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%255D%252C%2522id%2522%253A%2522clu8epe4z00053d5wpqy2p2r2%2522%252C%2522activeTabId%2522%253A%2522clu8epe4z00043d5wobnrq3u4%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

At path `/docs/intro` I have a link that is clickable, when it shouldn't be.

**Comment by Josh-Cena** [comment, marked as answer]
You can either wrap it in code, or use something like `<span>https://</span>example.com` to manually cut off a part of the URL. Neither are ideal I realize, but I recommend the first one.

**Reply by anatolykopyl** [reply]
Thank you for the fast reply!
Both are valid solutions for my case, but out of curiosity, what part of Docusaurus is responsible for this?
I found something called [extended url autolink](https://github.github.com/gfm/#extended-url-autolink) in the GFM spec, that seems like what is happening here. But the Docusaurus and MDX docs state that autolinks are disabled (the angle bracket kind).

Is there maybe a way to opt out the whole Docusaurus instance from url autolinks?

**Reply by Josh-Cena** [reply]
The thing that MDX calls "autolink" is just the angle bracket syntax, which confused me too. I don't think there's an MDX option to turn this plain-text link behavior off, although I would be happy to be proven wrong. This is purely implemented by remark-gfm I think, and Docusaurus does not play any part in it.
